src/nlp/apply_KB_unsupervised.py

```diff
@@ -0,0 +1,88 @@
+from keybert import KeyBERT
+import nltk
+from nltk.corpus import stopwords
+
+# nltk.download("stopwords")
+# nltk.download("punkt")
+
+# nltk.download("wordnet")
+# nltk.download("omw-1.4")
+from nltk import word_tokenize
+from nltk.stem import WordNetLemmatizer
+import string
+from transformers import AutoTokenizer, AutoModel
+import pandas as pd
+import transformers
+from transformers import pipeline
+
+# create also column for keywords from semisupervised model
+
+
+def KeywordExtraction(df):
+    tokenizer = AutoTokenizer.from_pretrained(
+        "models/nlp/unsupervised/model", model_max_lenght=512
+    )
+
+    # truncate all the text to 512 tokens
+
+    hf_model = pipeline(
+        "feature-extraction",
+        model="models/nlp/unsupervised/model",
+        tokenizer=tokenizer,  # "models/nlp/semi_supervised/model",
+    )
+
+    kw_model = KeyBERT(model=hf_model)
+    keywords = kw_model.extract_keywords(
+        df["transcription"],
+        keyphrase_ngram_range=(1, 2),
+        stop_words="english",
+        use_maxsum=True,
+        nr_candidates=20,
+        top_n=15,
+        use_mmr=True,
+        diversity=0.5,
+    )
```

Comment:
Wenn diese Funktion die gleich Funktion wie im KB unsupervised file ist, dann sollte sie in den utils file kommen und nur noch einmal existieren